CardinalityEstimation/ICardinalityEstimator.cs

```diff
@@ -42,5 +42,7 @@ public interface ICardinalityEstimator<in T>
         /// </summary>
         /// <returns>The estimated count of unique elements</returns>
         ulong Count();
+
+        ulong Size { get; }
```

Comment:
Let's rename this, how about "CountElementsAdded" ? Also, add the following comment:

```
/// <summary>
///     Returns the number of times elements were added (including duplicates)
/// </summary>
/// <returns>The number of times <see cref="Add"/> was called</returns>
ulong CountElementsAdded { get; }
```